### DISCUSSION TITLE
.text-right is not working on the div where list of buttons are used as children elements

### DISCUSSION THREAD

**Discussion by dhi-heraizen** [twbs/bootstrap#40717] [General]
.text-right is not working on the div where list of buttons are used as children elements when I upgraded to bootstrap 5,
but it was working in bootstrap 4.

```
<div class="col-md-8 col-sm-8 text-right ">
                        <button class="btn submit-button">
                           submit
                        </button>
                        <button class="btn reset-button">
                           Reset
                        </button>
 </div>
```

What should the right way to align div elements to the right.

**Comment by julien-deramond** [comment]
Hello @dhi-heraizen 
`.text-right` must be replaced by `.text-end` when migrating from Bootstrap v4 to v5 (source: https://getbootstrap.com/docs/5.3/migration/#utilities-3)
